src/components/DisplayToy.js

```diff
@@ -4,29 +4,32 @@ import { Container, Button } from 'react-bootstrap';
 import { useDispatch, useSelector } from 'react-redux';
 import { deleteToy } from '../actions/deleteToy.js';
 import { selectToy } from '../selectors/toySelectors';
-import { useHistory } from 'react-router';
+import { useParams } from 'react-router-dom';
+import { Redirect } from 'react-router-dom';
 
-const DisplayToy = (props) => {
+const DisplayToy = () => {
   const dispatch = useDispatch();
+  const { id } = useParams();
 
-  const toy = useSelector(selectToy(props.match.params.id));
+  const toy = useSelector(selectToy(id));
 
-  const history = useHistory();
-
-  return (
-    <Container>
-      <div>
-        <h1 className="header text-center">{toy.name}</h1>
-        <img src={toy.image_url} alt="toy-image" width="300" height="300" />
-        <p>{toy.description}</p>
-        <Button onClick={() => dispatch(deleteToy(toy.id, history))}>
-          Delete Toy
-        </Button>
-        <ReviewsContainer toy={toy} />
-      </div>
-      )
-    </Container>
-  );
+  if (toy)
+    return (
+      <Container>
+        <div>
+          <h1 className="header text-center">{toy.name}</h1>
+          <img src={toy.image_url} alt="toy-image" width="300" height="300" />
+          <p>{toy.description}</p>
+          <Button onClick={() => dispatch(deleteToy(toy.id))}>
+            Delete Toy
+          </Button>
+          <ReviewsContainer toy={toy} />
+        </div>
+      </Container>
+    );
+  else {
+    return <Redirect to="/toys" />;
+  }
```

Comment:
If you're leaving as is, this looks a little weird that there's only one set of {}, I'd expect to see 2 sets of {}, one for the `if`, the other for the `else`.

What's there now:
```
if (toy)
  ...
else {
  ...
}
```

...I'd propose to make things more consistent:

```
if (toy) {
 ...
} else {
  ...
};
```

Reply:
I am going to make the change you suggested because a one liner looks a lot cleaner for sure. Thanks!